version.php

```diff
@@ -23,7 +23,7 @@
  */
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2024100901;
+$plugin->version   = 2024120900;
 $plugin->release   = 2024100901;
```

Comment:
can we please bump the release version at the same time?

Reply:
Thanks @danmarsden 

I've bumped the release version as well